### ISSUE TITLE
User defined unit default unit

### ISSUE THREAD

**Issue by jukti3742** [connorferster/forallpeople#54]
Hi. First of all thanks a lot for all the good works you have done. While using **_forallpeople_**, I need the **output of force** in one specific unit i.e **kN**. But the auto-prefixing feature doesn't allow that. Like you have already mentioned it rounds up to the nearest 3rd power of ten. 
![1](https://user-images.githubusercontent.com/74715775/177048392-fe608c4e-70bd-44b9-9879-0d3330b53a16.png)
![2](https://user-images.githubusercontent.com/74715775/177048409-f238b95d-bf6e-4479-8d23-303b7e7aaff9.png)

I found one `.to` method. But it also does not seem to work well. Or may be I am wrong somewhere. But I tried.
![3](https://user-images.githubusercontent.com/74715775/177048501-aae69a9d-86ba-40cd-9509-e2323a0c6527.png)
 Moreover, even if the `.to` method works, there will be a lot of extra work to write one more line code after each of the **force variables**. So, I would like to know if there is any way to work around the problem.

**Comment by connorferster** [comment]
I totally agree. It would be good to set a user defined default unit in the environment.

**Comment by SpazzNon** [comment]
Hi, what is the solve? sorry about my english. I want define all my units of pression in MPa :C, is this possible?
